op-e2e/config/init.go

```diff
@@ -329,7 +329,7 @@ func initAllocType(root string, allocType AllocType) {
 			mtx.Unlock()
 
 			// This needs to be updated whenever the latest hardfork is changed.
```

Comment:
This seems very brittle given the comment. Do we want a `genesis.LatestHardfork` or something like that?

Reply:
Oops, this is what's done in https://github.com/ethereum-optimism/optimism/pull/14701.

Reply:
Yes that is a better solution. We see even more failures on that PR, though. Let's continue conversation over there.